java/bench/spark/pom.xml

```diff
@@ -135,6 +134,21 @@
       <groupId>org.scala-lang</groupId>
       <artifactId>scala-library</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-core</artifactId>
+      <version>2.10.0</version>
```

Comment:
Can you please define a variable to hold the jackson version?

Reply:
Sure!